### PR TITLE
Update Community slack information and resources 

### PR DESCRIPTION
**Pull request by lorikrell**
## This PR is a:

- [ ] New topic
- [X] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will...

Update and add links for the resources page of new Community content, Slack channels per request
Add tip note to Contributing page for joining the Slack channels
Link version updates (minor edit)

## Additional information

List all affected URLs 

- https://devdocs.magento.com/community/resources/resources.html
- https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html
- https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html

whatsnew
Find new [Community Resources](https://devdocs.magento.com/community/resources/resources.html) and Slack information for Contributors and Maintainers.

### CODE REVIEW

File: community/resources/best-practices.md [hunk 1]
```diff
@@ -21,16 +21,16 @@ They are available in DevDocs because the content has been well received within
...
-- [Programming Best Practices](https://devdocs.magento.com/guides/v2.2/ext-best-practices/extension-coding/common-programming-bp.html)
-- [Observers Best Practices](https://devdocs.magento.com/guides/v2.2/ext-best-practices/extension-coding/observers-bp.html)
-- [Theme development best practices](https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/theme-best-practice.html)
-- [Performance Best Practices](https://devdocs.magento.com/guides/v2.2/performance-best-practices/)
-- [Best Practices for Store Configuration](https://devdocs.magento.com/guides/v2.2/cloud/configure/configure-best-practices.html) for Commerce Cloud
-- [Deployment process](https://devdocs.magento.com/guides/v2.2/cloud/reference/discover-deploy.html) for Commerce Cloud
+- [Programming Best Practices](https://devdocs.magento.com/guides/v2.3/ext-best-practices/extension-coding/common-programming-bp.html)
+- [Observers Best Practices](https://devdocs.magento.com/guides/v2.3/ext-best-practices/extension-coding/observers-bp.html)
+- [Theme development best practices](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/theme-best-practice.html)
```
Comment (shrielenee) [inline community/resources/best-practices.md]: Capitalize this like the rest

File: community/resources/best-practices.md [hunk 2]
```diff
@@ -21,16 +21,16 @@ They are available in DevDocs because the content has been well received within
...
+- [Performance Best Practices](https://devdocs.magento.com/guides/v2.3/performance-best-practices/)
+- [Best Practices for Store Configuration](https://devdocs.magento.com/guides/v2.3/cloud/configure/configure-best-practices.html) for Commerce Cloud
+- [Deployment process](https://devdocs.magento.com/guides/v2.3/cloud/reference/discover-deploy.html) for Commerce Cloud
```
Comment (shrielenee) [inline community/resources/best-practices.md]: cap?

File: community/resources/resources.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ This Community collection of resources links to blog posts, podcasts, presentati
 
 New items will be added incrementally, and older links rotated out. Keep an eye on this space for the latest community resources!
 
- Another community-based resource, focused on describing the Magento 2 Environment is the [Magento 2 Awesome List][0]
+Another community-based resource, focused on describing the Magento 2 Environment is the [Magento 2 Awesome List][0]
```
Comment (shrielenee) [inline community/resources/resources.md]:
```suggestion
Another community-based resource, focused on describing the Magento 2 environment, is the [Magento 2 Awesome List][0].
```

File: guides/v2.2/contributor-guide/contributing.md [hunk 4]
```diff
@@ -19,6 +19,9 @@ The following topics are included in this guide:
 - [Labels applied by the Community Engineering Team](#labels)
 - [Contribution awards and points](#points)
 
+{:.bs-callout .bs-callout-tip}
+Connect with Magento Contributors and Maintainers, learning more about Magento contributions and special projects.  Click here to join our [Slack workspace](https://tinyurl.com/engcom-slack). Join us in [#general](https://magentocommeng.slack.com/messages/C4YS78WE6), follow the [#announcements](https://magentocommeng.slack.com/messages/C7FA71S3V), and browse for more [channels](https://devdocs.magento.com/community/resources/resources.html#community-engineering-slack)!
```
Comment (shrielenee) [inline guides/v2.2/contributor-guide/contributing.md]:
```suggestion
Connect with Magento Contributors and Maintainers to learn more about Magento contributions and special projects. Join us in our [Slack workspace](https://tinyurl.com/engcom-slack), in the [#general](https://magentocommeng.slack.com/messages/C4YS78WE6) channel, follow the [#announcements](https://magentocommeng.slack.com/messages/C7FA71S3V), and browse for more [channels](https://devdocs.magento.com/community/resources/resources.html#community-engineering-slack)!
```

File: guides/v2.3/contributor-guide/contributing.md [hunk 5]
```diff
@@ -18,6 +18,9 @@ The following topics are included in this guide:
 - [Labels applied by the Community Engineering Team](#labels)
 - [Contribution awards and points](#points)
 
+{:.bs-callout .bs-callout-tip}
+Connect with Magento Contributors and Maintainers, learning more about Magento contributions and special projects. Click here to join our [Slack workspace](https://tinyurl.com/engcom-slack). Join us in [#general](https://magentocommeng.slack.com/messages/C4YS78WE6), follow the [#announcements](https://magentocommeng.slack.com/messages/C7FA71S3V), and browse for more [channels](https://devdocs.magento.com/community/resources/resources.html#community-engineering-slack)!
```
Comment (shrielenee) [inline guides/v2.3/contributor-guide/contributing.md]:
```suggestion
Connect with Magento Contributors and Maintainers to learn more about Magento contributions and special projects. Join us in our [Slack workspace](https://tinyurl.com/engcom-slack), in the [#general](https://magentocommeng.slack.com/messages/C4YS78WE6) channel, follow the [#announcements](https://magentocommeng.slack.com/messages/C7FA71S3V), and browse for more [channels](https://devdocs.magento.com/community/resources/resources.html#community-engineering-slack)!
```

**Comment by lorikrell** [comment]
running tests

**Pull request merged**